packages/vuetify/src/mixins/data-iterable.js

```diff
@@ -227,6 +227,11 @@ export default {
   },
 
   watch: {
+    items () {
+      if (this.pageStart >= this.itemsLength) {
```

Comment:
This fix somehow breaks external pagination where total number is greater then length of items.
In my code items and total are computed fields and this code resets pagination every time.

Reply:
Ugh, I was afraid of something breaking. Can you create a codepen reproduction?

Reply:
> This fix somehow breaks external pagination where total number is greater then length of items.
> In my code items and total are computed fields and this code resets pagination every time.

Had similar issue at server-side pagination exactly after this commit. 

Was my fault (I reset total items counter to 0 before requesting server). Your code **works as expected**.